### ISSUE TITLE
Select compatibility in recent versions

### ISSUE THREAD

**Issue by remixer-dec** [c-smile/sciter-js-sdk#88]
After updating from 4.4.6.6 to 4.4.7.2, I noticed that < select > started to display incorrectly within elements attached to Vue. 

![Screenshot](https://i.imgur.com/bmSfV5q.png)
Here is the demo code
```html
<!DOCTYPE html>
<html>
<head>
  <meta charset="UTF-8">
  <title>Select test</title>
</head>
<body>
  <div id="app">
    <select>
      <option>Test 1</option>
      <option>Test 2</option>
    </select>
  </div>
  <script src="https://cdn.jsdelivr.net/npm/vue@2.6.12/dist/vue.min.js"></script>
  <script>
    const APP = new Vue({el: '#app', data: {}})
  </script>
</body>
</html>
```

**Comment by c-smile** [comment]
No idea yet why Vue modifies content of `<select>` (and `<textarea>` for that matter)

**Comment by remixer-dec** [comment]
Maybe because of [this](https://vuejs.org/v2/guide/components.html#DOM-Template-Parsing-Caveats)? Not sure what's wrong with `<textarea>`, looks fine to me. Both of these tags are mentioned in Vue's source [here](https://github.com/vuejs/vue/blob/v2.6.12/src/platforms/web/runtime/directives/model.js#L22) and [here](https://github.com/vuejs/vue/blob/v2.6.12/src/platforms/web/util/attrs.js#L10) and in [Vue3](https://github.com/vuejs/vue-next/blob/v3.0.11/packages/runtime-dom/src/directives/vModel.ts#L276). Also, this bug appeared in 4.4.6.6 with the `<option>` tags created with `v-for` loop, and a way to fix it was to put an extra `<option>` tag before the loop. With 4.4.7.* everything with select tag seems to be broken.

But in general, looks like SciterJS has pretty good compatibility with Vue (2) and to get Vue3 to work [this](https://gist.github.com/remixer-dec/341c0b707aae918f4c45dc46e2fc527b) seems to be enough.

**Comment by vrimar** [comment]
I am also facing the same issue with Mithril.js. This was working in earlier version but broken since 4.4.6.6.

The code used:
```
<html>
<head>
  <script src="lib/mithril.min.js"></script>
  <script>
    m.mount(document.getElementById("mithril"), {
      view: () => m("select", [
        m("option", "First"),
        m("option", "Second")
      ])
    });
  </script>
</head>

<body>
  <div id="mithril"></div>
  <select>
    <option>First</option>
    <option>Second</option>
  </select>
</body>
</html>

```

As you can see in the screenshot, mithril.js is rendering the children at the end instead of into the "popup.list".

![image](https://user-images.githubusercontent.com/5660247/117194734-709c2f80-adb2-11eb-86d7-cd6b4b94e72a.png)

Any ideas?

**Comment by c-smile** [comment]
Should be fixed in 4.4.8.7
